src/config_trapdoor.c

```diff
@@ -798,7 +798,7 @@ TbBool parse_trapdoor_trap_blocks(char *buf, long len, const char *config_textna
               k = atoi(word_buf);
               if (k >= 0)
               {
-                  gameadd.trap_stats[i].light_radius = k;
+                  gameadd.trap_stats[i].light_radius = k << 8; //keep it the same as the object light radius
```

Comment:
maybe a * COORD_PER_STL is more readable then a bitshift

Reply:
it's more readable for sure,... but do you think that is what it represents? Would make sense I guess.

Reply:
yeah looks like radius stores full coord, and the config does it in subtiles

Reply:
ok, good. I updated.